browser_tests/tests/menu.spec.ts

```diff
@@ -178,6 +178,75 @@ test.describe('Menu', () => {
       await comfyPage.menu.topbar.triggerTopbarCommand(['ext', 'foo-command'])
       expect(await comfyPage.getVisibleToastCount()).toBe(1)
     })
+
+    test('Can navigate Theme menu and switch between Dark and Light themes', async ({
+      comfyPage
+    }) => {
+      // Open the topbar menu
+      await comfyPage.menu.topbar.openTopbarMenu()
+      const menu = comfyPage.page.locator('.comfy-command-menu')
+      await expect(menu).toBeVisible()
+
+      // Navigate to Theme menu item and open its submenu
+      const themeMenuItem = comfyPage.page.locator(
+        '.p-menubar-item-label:text-is("Theme")'
+      )
```

Comment:
Could you extract a Fixture for the menus?

Reply:
(and maybe the submenus, but those would probably be in scope for the Menu fixture itself)

Reply:
I usually consider explicit `.locator`s in tests to be a minor code smell.